portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/announcements/AnnouncementsAdmin.testcase

```diff
@@ -0,0 +1,29 @@
+definition {
+
+	property testray.main.component.name = "Announcements";
+
+	setUp {
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		JSONAnnouncement.deleteAnnouncement(
+			groupName = "Guest",
+			siteURLKey = "guest");
```

Comment:
Please add a conditional here which satisfies:
- If there still exists an announcement entry, delete it